### PR TITLE
Resolving FogBugz 1269773, 4,  & 5, plus 1284567

### PR DESCRIPTION
**Pull request by ValGrimm-U3D**
# **Please read the [Contributing guide](CONTRIBUTING.md) before making a PR.**

* Read the [Graphics repository & Yamato FAQ](http://go/graphics-yamato-faq).

### Checklist for PR maker
- [Y ] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [ N/A] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [N/A ] Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR. If you do add documentation, make sure to add the relevant Graphics Docs team member as a reviewer of the PR. If you are not sure which person to add, see the [Docs team contacts sheet](https://docs.google.com/spreadsheets/d/1rgUWWgwLFEHIQ3Rz-LnK6PAKmbM49DZZ9al4hvnztOo/edit#gid=1058860420).
- [N/A ] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?
Resolving FogBugz 1269773,1269774, 1269775, 1284567

---
### Testing status
Describe what manual/automated tests were performed for this PR

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.shadergraph/Documentation~/index.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ Shader Graph enables you to build shaders visually. Instead of writing code, you
...
+Shader Graph is available through the Package Manager window in supported versions of the Unity Editor. If you install a prebuilt Scriptable Render Pipeline (SRP) such as the [Universal Render Pipeline](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest) (URP) or the [High Definition Render Pipeline](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@latest) (HDRP), Unity automatically installs Shader Graph in your project.
 
-Avoid installing or updating Shader Graph independently of the prebuilt SRP packages. Shader Graph builds shaders that are compatible with the URP and HDRP, but they are not compatible with the built-in renderer.
+Shader Graph is not compatible with the Built-In Render Pipeline.
```
Comment (marctem) [inline com.unity.shadergraph/Documentation~/index.md:11]: This isn't true anymore! =)

**Review by marctem — CHANGES_REQUESTED** [review]

File: com.unity.shadergraph/Documentation~/index.md (hunk 1 above)
Comment (jessebarker) [reply com.unity.shadergraph/Documentation~/index.md:11]: I was waiting for the merge to land before commenting :-).  Good catch.

Comment (ValGrimm-U3D) [reply com.unity.shadergraph/Documentation~/index.md]: Have fixed this.

**Review by CertifiedRice — APPROVED** [review]

**Comment by ValGrimm-U3D** [comment]
It seems that I need @marctem to approve as he requested changes, which I did make.

**Review by marctem — APPROVED** [review]

**Pull request merged**